scripts/rollup/results.json

```diff
@@ -0,0 +1,1446 @@
+{
```

Comment:
We got rid of this file on master, I think? It isn't supposed to be checked in anymore. Only runs on CI.

Reply:
I have deleted the file.